docs/bootstrap.md

```diff
@@ -1,35 +1,56 @@
-# Bootstrapping a Kubernetes cluster
+# Bootstrapping Dragonfly on Digital Ocean
 
-## Create namespaces
+## Prerequisites
 
-```bash
-kubectl apply -f kubernetes\manifests\cert-manager\namespace.yaml
-kubectl apply -f kubernetes\manifests\discord\namespace.yaml
-kubectl apply -f kubernetes\manifests\dragonfly\namespace.yaml
+You will need:
+
+* [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/) configured with a...
+* [Digital Ocean Personal Access Token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) with at least the following scopes:
+  * kubernetes (2): create, delete
+  * If replacing an existing cluster:
+    * load\_balancer (1): delete
+* PWD set to the root of this repo
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [`helm`](https://helm.sh/docs/intro/install/)
+
+## Create the cluster in Digital Ocean
+```
+doctl k8s cluster create \
+  <cluster-name> \
+  --size s-2vcpu-2gb \
+  --count 2 \
+  --region nyc3
 ```
 
-## Install the Helm Chart to get all the dependencies
+Adjust `size` and `count` as needed. See [`doctl kubernetes cluster create
+docs`](https://docs.digitalocean.com/reference/doctl/reference/kubernetes/cluster/create/)
+for more options.
+
+## Apply `cert-manager` CRDs

```

Comment:
`suggestion(blocking)`: Add a step to make sure the kube context is set to the new cluster.

Reply:
Creating the cluster using `doctl` does this implicitly. Could add a note to reassure readers that this is happening, though the `doctl` output does say this I think.